releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -2,7 +2,7 @@
   "109939": {
     "commit": "802983e7142b3a1edf300712cfe5dca5dafbfd49",
     "text": "Added support for building Windows kube-proxy container image.\nA container image for kube-proxy on Windows can now be built with the command\n`make release-images KUBE_BUILD_WINDOWS=y`.\nThe Windows kube-proxy image can be used with Windows Host Process Containers.",
-    "markdown": "Added support for building Windows kube-proxy container image.\n  A container image for kube-proxy on Windows can now be built with the command\n  `make release-images KUBE_BUILD_WINDOWS=y`.\n  The Windows kube-proxy image can be used with Windows Host Process Containers. ([#109939](https://github.com/kubernetes/kubernetes/pull/109939), [@claudiubelu](https://github.com/claudiubelu)) [SIG Windows]",
+    "markdown": "Added support for building Windows kube-proxy container image.\n  A container image for kube-proxy on Windows can now be built with the command\n  `make release-images KUBE_BUILD_WINDOWS=y`.\n  The Windows kube-proxy image can be used with Windows Host Process Containers. ([#109939](https://github.com/kubernetes/kubernetes/pull/109939), [@claudiubelu](https://github.com/claudiubelu))",
```

Comment:
Ran with no `--repo` flag:

```
 go run ./cmd/krel release-notes --create-draft-pr --fork=npolshakova --fix --tag  v1.31.0-alpha.3 --list-v2 --log-level trace
```

It looks like the sig labels are still being removed